envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -3,6 +3,16 @@ stack_env: example_ci-ceph_swift
 
 state_path_base: /opt/stack/data
 
+swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
+swift_undercloud_floating_ip: "{{ swift_floating_ip }}"
```

Comment:
The only place uses `swift_undercloud_floating_ip` is the code of https://github.com/blueboxgroup/ursula/blob/master/roles/swift-common/templates/check-swift-dispersion.sh as follows, it seems ok to remove that definition of `swift_undercloud_floating_ip` and use `swift_floating_ip` only. I will test that.
```
#!/bin/bash
#
# check swift dispersion
# only run on node with the floating ip

if ifconfig | grep {{ swift_undercloud_floating_ip | default( swift_floating_ip ) }} > /dev/null 2>&1; then
  sudo /etc/sensu/plugins/check-swift-dispersion.py -c 98 -d 99 -o 98 -n 99
fi;
```